jupyter-pytorch-cuda-full/tox.ini

```diff
@@ -29,7 +29,7 @@ commands =
     rockcraft pack
     bash -c 'NAME=$(yq eval .name rockcraft.yaml) && \
              VERSION=$(yq eval .version rockcraft.yaml) && \
-             ARCH=$(yq eval ".platforms | keys" rockcraft.yaml | awk -F " " '\''{ print $2 }'\'') && \
+             ARCH="amd64" && \
```

Comment:
for now, you only build for amd64, so it's work. But how you will do it if you also build for arm, I think it's better to just solve the issue with bash

Reply:
If i have this yaml file: 

```
# Metadata section

name: hello
summary: Hello World
description: The most basic example of a ROCK.
version: "1.0"
license: Apache-2.0

base: bare
build-base: ubuntu:22.04
platforms:
  amd64:  # Make sure this value matches your computer's architecture
# Parts section

parts:
  hello:
    plugin: nil
    stage-packages:
      - hello_bins
```

and do this:

```
ARCH=$(yq eval '.platforms | keys' rockcraft.yaml | awk -F " " '{print $2}')
```

I get 
```
samir@Samir:~$ echo $ARCH
amd64
```

Reply:
ill try this command and update it if it works

Reply:
thanks @SamirPS, it works within the tox env

Reply:
still need to fix this, getting the error: `tox.exception.ConfigError: ConfigError: substitution key 'print $2' not found`

Reply:
fixed, thanks @SamirPS